config/mbed/CMakeLists.txt

```diff
@@ -132,34 +138,31 @@ else()
     set(CHIP_PROJECT_CONFIG "")
 endif()
 
+if (${CMAKE_BUILD_TYPE} STREQUAL "debug")
+    set(CONFIG_DEBUG "y")
```

Comment:
That's a tricky one, mbed has three configuration: 
- `debug` => compilation without optimization and assertions
- `develop` => Compilation with optimization and assertions 
- `release` => compilation with optimization and without assertions

Reply:
Right, but CHIP has only two states is_debug on or off, which means development features are enabled or disabled. 
The question is how to treat the development state. If it uses the optimization, it is more suited to is_debug=off.